ipfs-cluster-ctl/main.go

```diff
@@ -412,10 +409,17 @@ func parseFlag(t int) cli.IntFlag {
 	}
 }
 
-func walkCommands(cmds []cli.Command) {
+func walkCommands(cmds []cli.Command, parentHelpName string) {
 	for _, c := range cmds {
-		fmt.Println(c.HelpName)
-		walkCommands(c.Subcommands)
+		var h string
+		// Sometimes HelpName is empty :S
+		if h = c.HelpName; h != "" {
```

Comment:
This block is a bit confusing to me, one of the below might be more clear --

```go
h := c.HelpName
if h == "" {
    h = fmt.Sprintf("%s %s", parentHelpName, c.FullName())
}
fmt.Println(h)
```

or, if you prefer

```go
var h string
if c.HelpName != "" {
    h = c.HelpName
} else {
    h = fmt.Sprintf("%s %s", parentHelpName, c.FullName())
}
fmt.Println(h)
```

Up to you of course, just my preference.